chromium_src/components/autofill/core/browser/autofill_browser_util.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#define IsFormMixedContent IsFormMixedContent_ChromiumImpl
+#include "src/components/autofill/core/browser/autofill_browser_util.cc"
+#undef IsFormMixedContent
+
+namespace {
+constexpr char kOnionDomain[] = "onion";
```

Comment:
this is already in net/base/url_utils.h

Reply:
actually it's in net/base/url_util.cc so it should move to net/base/url_util.h so we don't duplicate it here